keyboards/ashpil/modelm_usbc/README.md

```diff
@@ -0,0 +1,2 @@
+# USB C Model M
```

Comment:
Only thing I'd still like to see here is for this file to be formatted like the [AVR template](https://github.com/qmk/qmk_firmware/blob/85cccdb6cc8a02cc103f9e5fd2829355e61f20e3/quantum/template/avr/readme.md). Everything else looks good.

Reply:
Done, thanks!